resources/assets/js/admin/directives/reset.js

```diff
@@ -18,16 +18,23 @@
                             Toast.push('success', $translate.instant('toast.contents.reset.success'), '');
                             localStorage.clear();
 
-                            setTimeout(function() {
+			    // Stop the screen capture if setting enabled at the school.
+
+                            if(config['video-do-recording']) {
+                                var name = prompt($translate.instant('toast.contents.reset.video.prompt'));
+                            	Recorder.stopRecording(name);
+                            }
+			    
+			    setTimeout(function() {
                                 window.location.reload();
-                            }, 500);
+                            }, 2000);
```

Comment:
Is this for debugging purposes (ie. to see whether anything fails before resetting) or is this desired behaviour?

Reply:
Current setting of 500ms does not allow the succes/error toasts to pop up.
2000ms is the value where it doesn't feel slow and it is possible to see if the video was saved or not.

An alternative that might be to prefer is to show the toasts after the window reload.

Reply:
Makes sense!